builder/azure/arm/config.go

```diff
@@ -623,6 +624,12 @@ type Config struct {
 	// Requires enabling encryption at host in the Subscription read more [here](https://learn.microsoft.com/en-us/azure/virtual-machines/disks-enable-host-based-encryption-portal?tabs=azure-powershell)
 	EncryptionAtHost *bool `mapstructure:"encryption_at_host" required:"false"`
 
+	// Specifies the SKU to use in the Public IP Address created to connect to the build Virtual machine.
+	// Can only be `Basic`, or `Standard`. Currently basic is the default if none is set.
+	// On 31 March 2025 Azure will remove the ability to create `Basic` SKU public IPs, before then the plugin will be updated to create a Standard public IP by default
+	// You can read more about public IP skus [here](https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/public-ip-addresses#sku)
```

Comment:
Slight reword to match the wording for similar SKU options and replace "read" with something a bit more inclusive. 
```suggestion
	// Specify the Public IP Address SKU for the public IP used to connect to the build Virtual machine.
	// Valid values are `Basic` and `Standard`. The default value is `Basic`.
	// On 31 March 2025 Azure will remove the ability to create `Basic` SKU public IPs, before then the plugin will be updated to create a Standard public IP by default
	// You can learn more about public IP SKUs [here](https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/public-ip-addresses#sku)
```

On a side note, I see the SKUs are capitalized should we be checking for this? Or is the upstream API agnostic to casing?